crates/sqlexec/src/background_jobs.rs

```diff
@@ -0,0 +1,326 @@
+use std::{
+    collections::HashMap,
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc, Mutex,
```

Comment:
I typically prefer to use the parking_lot mutex. Most of the times it's faster, and no need to `unwrap` after `lock`.

Reply:
Will update using that.